messages/de-DE.json

```diff
@@ -0,0 +1,384 @@
+{
+  "Header": {
+    "groups": "Gruppen"
+  },
+  "Footer": {
+    "madeIn": "Entwickelt in Montréal, Québec, Kanada",
+    "builtBy": "Erstellt von <author>Sebastien Castiel</author> und <source>Mitwirkenden</source>"
+  },
+  "Expenses": {
+    "title": "Ausgaben",
+    "description": "Hier sind die Ausgaben, die du für deine Gruppe erstellt hast.",
+    "create": "Ausgabe hinzufügen",
+    "createFirst": "Erstelle die Erste",
+    "noExpenses": "Deine Gruppe hat noch keine Ausgaben.",
+    "exportJson": "Als JSON exportieren",
+    "searchPlaceholder": "Suche nach einer Ausgabe…",
+    "ActiveUserModal": {
+      "title": "Wer bist du?",
+      "description": "Sag uns, welcher Teilnehmer du bist, damit wir die Informationen, die wir dir zeigen, auf dich zuschneiden können.",
```

Comment:
Weirdly convoluted in my opinion. I would rephrase to "Sag uns, welcher Teilnehmer du bist, um die angezeigten Informationen auf dich anzupassen." or similar.